cli/i18n/I18nData.php

```diff
@@ -329,5 +329,4 @@ public function getReferenceLanguage(): array {
 	private function getFilenamePrefix(string $key): string {
 		return preg_replace('/\..*/', '.php', $key) ?? '';
 	}
-
```

Comment:
This strikes me as a potentially somewhat unfortunate thing to enforce.

Reply:
Ok, I will exclude a few more rules

Reply:
Fixed